### PR TITLE
Update Activity

### PR DESCRIPTION
**Pull request by ifndefdeadmau5**
- Added ‘ComponentWillReceiveProps’ to NewRequests so that component
can pull data whenever page shows up.
- Since concat method doesn’t manipulate original array itself, re-assigned the
return value.

### CODE REVIEW

File: src/components/Activity/NewRequests.js [hunk 1]
```diff
@@ -28,6 +28,10 @@ class NewRequests extends Component {
     MatchUtil.getActivityList(this.onRequestCallback.bind(this));
   }
 
+  componentWillReceiveProps(props) {
+    MatchUtil.getActivityList(this.onRequestCallback.bind(this));
+  }
```
Comment (SarahhhC) [inline src/components/Activity/NewRequests.js:33]: Was there error in activity?

Comment (ifndefdeadmau5) [reply src/components/Activity/NewRequests.js:33]: It didn't refresh the data when activity page shows up

**Comment by MechanicKim** [comment]
I reviewed it. and... @SarahhhC ?

**Comment by SarahhhC** [comment]
LGTM 👍

**Pull request merged**